test/EndToEndTests/Tests/Client/Build.Desktop/TransportLayerTests/RequestMessageArgsTests.cs

```diff
@@ -41,7 +41,7 @@ public void AddCustomHeader()
                 (args) =>
                 {
                     args.Headers.Add(headerName, headerValue);
-                    return new HttpWebRequestMessage(args);
+                    return new Microsoft.OData.Client.HttpClientRequestMessage(args);

```

Comment:
Is it possible to add `using Microsoft.OData.Client.HttpClientRequestMessage` since this is used in multiple places?

Reply:
There will be a conflict.. There is a similar name in a test project

Reply:
It will conflict with a similar class name in a test project that is why I decided to go with the full name.